roles/debian/wazuh/tasks/main.yml

```diff
@@ -139,6 +139,10 @@
   ignore_errors: true
   changed_when: false
 
+- name: Set fact if wazuh-manager service exists
+  ansible.builtin.set_fact:
+    wazuh_manager_exists: "'wazuh-manager.service' in wazuh_service.stdout"
```

Comment:
Pretty sure this will just set a string with the content `'wazuh-manager.service' in wazuh_service.stdout`? Does it need to be in `{{ }}` at least?